eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -63,6 +63,7 @@ jobs:
       variables:
         VcpkgDependencies: curl[winssl] libxml2
         VCPKG_DEFAULT_TRIPLET: 'x64-windows-static'
+        Package.EnableSBOMSigning: true
```

Comment:
Is this required?

Reply:
No, but given they have us slapping this everywhere, I doubt it'll do anything in public by default.